CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [3.9.0](https://github.com/marklogic/mlxprs/releases/tag/v3.9.0)
+
+- New MarkLogic status view including high-level information about the currently connected MarkLogic server
```

Comment:
Include periods at the end. And I think it's worth calling out the activity bar icon explicitly here - it's a new, noticeable feature that should create some immediate interest.